tools/aws_lab_setup/provision_lab.yml

```diff
@@ -26,3 +44,11 @@
   gather_facts: no
   roles:
     - email
+
+- name: Setup Amazon S3 Website for Student Login
```

Comment:
I'd like this whole role pulled and submitted as a separate PR for consideration.